Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/audit/transform/PatientDiscoveryDeferredResponseAuditTransforms.java

```diff
@@ -0,0 +1,221 @@
+ *       derived from this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE UNITED STATES GOVERNMENT BE LIABLE FOR ANY
+ * DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package gov.hhs.fha.nhinc.patientdiscovery.audit.transform;
+
+import com.services.nhinc.schema.auditmessage.AuditMessageType;
+import gov.hhs.fha.nhinc.audit.AuditTransformsConstants;
+import gov.hhs.fha.nhinc.common.nhinccommon.AssertionType;
+import gov.hhs.fha.nhinc.common.nhinccommon.HomeCommunityType;
+import gov.hhs.fha.nhinc.common.nhinccommon.NhinTargetCommunityType;
+import gov.hhs.fha.nhinc.common.nhinccommon.NhinTargetSystemType;
+import gov.hhs.fha.nhinc.common.nhinccommon.UserType;
+import gov.hhs.fha.nhinc.connectmgr.ConnectionManagerException;
+import gov.hhs.fha.nhinc.nhinclib.NhincConstants;
+import gov.hhs.fha.nhinc.patientdiscovery.MessageGeneratorUtils;
+import gov.hhs.fha.nhinc.patientdiscovery.parser.PRPAIN201306UV02Parser;
+import gov.hhs.fha.nhinc.properties.PropertyAccessException;
+import gov.hhs.fha.nhinc.properties.PropertyAccessor;
+import gov.hhs.fha.nhinc.transform.audit.AuditDataTransformHelper;
+import java.lang.management.ManagementFactory;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Properties;
+import javax.xml.bind.JAXBException;
+import org.apache.log4j.Logger;
+import org.hl7.v3.MCCIIN000002UV01;
+import org.hl7.v3.PRPAIN201306UV02;
+
+/**
+ *
+ * @author achidamb
+ */
+public class PatientDiscoveryDeferredResponseAuditTransforms extends AbstractPatientDiscoveryAuditTransforms<PRPAIN201306UV02, 
+    MCCIIN000002UV01> {
+
+    private static final Logger LOG = Logger.getLogger(PatientDiscoveryDeferredResponseAuditTransforms.class);
+
+    public PatientDiscoveryDeferredResponseAuditTransforms() {
+        
+    }
+
+    @Override
+    protected AuditMessageType getParticipantObjectIdentificationForRequest(PRPAIN201306UV02 request,
+        AssertionType assertion, AuditMessageType auditMsg) {
+        auditMsg = getPatientParticipantObjectIdentificationForResponse(request, auditMsg);
+        try {
+            auditMsg = getQueryParamsParticipantObjectIdentificationForResponse(request, auditMsg);
+        } catch (JAXBException ex) {
+            if (LOG.isDebugEnabled()) {
```

Comment:
Remove this "if" statement.  That said, here are a few things for future reference:

1. You use "is[Level]Enabled" to check that specific level (e.g., if you're going to log trace, you'd check "isTraceEnabled()", not "isDebugEnabled").
2. There is no "isErrorEnabled" -- unless you disable logging altogether, it's always enabled.  I suppose you could use "isEnabledFor(Level.ERROR)", but you'd only be checking if logging is on at all.
3. As a rule of thumb, unless the logging operation is extremely heavy (e.g., transforming a large object for the trace logs), you don't want to check if "Info" level or lower are enabled, only higher (debug, trace).